contracts/deposit-contract/sendDepositTx/sendDeposits_test.go

```diff
@@ -31,10 +30,14 @@ func generateValidators(count int64) map[string]*prysmKeyStore.Key {
 
 }
 
-//should I explicitly return err here?
+func init() {
+	logrus.SetLevel(logrus.DebugLevel)
+	logrus.SetOutput(ioutil.Discard)
+}
+
 func sendDeposits(testAcc *contracts.TestAccount, validatorKeys map[string]*prysmKeyStore.Key,
 	numberOfDeposits int64) {
-	depositAmountInGwei := uint64(contracts.Amount32Eth.Int64())
+	depositAmountInGwei := contracts.Amount32Eth.Uint64()  //how to fix an error here?
```

Comment:
@nisdas  i followed your suggestion to use Uint64() here, I encounter the same error /home/ai/.cache/bazel/_bazel_ai/950cfae04b722934297cff87d8a55db5/sandbox/linux-sandbox/201/execroot/__main__/contracts/deposit-contract/sendDepositTx/sendDeposits_test.go:40:46: 
depositcontract.Amount32Eth.Uint64 undefined (type func() *big.Int has no field or method Uint64)